README.md

```diff
@@ -15,11 +15,31 @@ For application features, build instructions and user guides see the [srsRAN doc
 sudo snap install srsran
 ```
 
-### Run the EPC
+## Config files
```

Comment:
Config files and Commands should be under the `usage` section

Reply:
✅